test/example/stack-utils.cpp

```diff
@@ -0,0 +1,369 @@
+#include <cassert>
```

Comment:
Do tests have to be written in this way, meaning, C++? Can't we use wast format? Maybe we should come up with a new extension given that the semantics of this file will be different from that of wast, but still, wouldn't tests in wast format be more readable than this?

Reply:
We can't use wast tests yet because that would require being able to validate poppy IR, which will depend on these utilities. Wast tests also wouldn't be as good for these tests because they depend on being able to compare the identity of individual expressions, so it is easier to explicitly build them.